L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp3.cc

```diff
@@ -962,8 +947,15 @@ bool HGCalTriggerGeometryV9Imp3::validTriggerCell(const unsigned trigger_cell_id
 
 bool HGCalTriggerGeometryV9Imp3::disconnectedModule(const unsigned module_id) const {
   bool disconnected = false;
-  if (disconnected_layers_.find(layerWithOffset(module_id)) != disconnected_layers_.end())
+  HGCalModuleDetId id(module_id);
+  if (module_to_stage1_.find(packLayerSubdetWaferId(id.layer(), id.triggerSubdetId(), id.moduleU(), id.moduleV())) !=
```

Comment:
Shouldn't it be the opposite? ( `module_to_stage1_.find(...) == module_to_stage1_.end()`)

Reply:
Yes, thanks